src/cards/CardRequirement.ts

```
@@ -257,9 +256,6 @@ export class PartyCardRequirement extends CardRequirement {
     return this.party.toLowerCase();
   }
   public satisfies(player: Player): boolean {
-    if (player.game.turmoil !== undefined) {
```

This seems like it could lead to a bug. If calling to `satisfies` and `player.game.turmoil` is `undefined` this will throw an error versus returning `false`.

---

I reviewed the rest of these and this is the only change that stands out as a potential problem. Using a `Turmoil.ifTurmoilElse` here makes sense to me. Otherwise any caller to `satisfies` needs the turmoil expansion or a runtime error will be thrown.

---

Cards declared with a party requirement not declared with turmoil SHOULD throw an error, see?

---

Cards declared with a party requirement not declared to be compatible with Turmoil SHOULD throw an error.

---

I'll expand: cards that require Turmoil need to have one of two conditions: it needs to be part of the Turmoil manifest, or, part of another manifest specifying `compatibility: GameModule.Turmoil` so they're only included when game.turmoil is included.